README.md

```diff
@@ -187,7 +189,7 @@ python dinov2/run/eval/linear.py \
 ```
 
 We release the weights from evaluating the different models:
-
+<div align="center">
```

Comment:
Same comment as above on styling the table instead (and please keep a separating blank line).